docs/client_install.md

```diff
@@ -0,0 +1,47 @@
+<!---
+Licensed under Creative Commons Attribution 4.0 International License
+https://creativecommons.org/licenses/by/4.0/
+--->
+
+# Client-Only Installation
+
+Instructions in this document assume the environment variable
+`PDO_SOURCE_ROOT` points to the PDO source directory. Information about
+other [environment variables](environment.md) that control build and
+installation is described below.
+
+## Process Overview
+
+- Setup the basic development environment
+- Install required build dependencies
+- Set up environment variables to configure the build
+- Build the PDO package
+
+## <a name="environment">Setup Build Environment</a>
+
+On a minimal Ubuntu system, the following packages are required. Other
+distributions will require similar packages.
+
+```bash
+sudo apt install -y cmake curl git pkg-config unzip xxd libssl-dev build-essential
+sudo apt install -y swig python3 python3-dev python3-venv virtualenv
+sudo apt install -y liblmdb-dev libprotobuf-dev libsecp256k1-dev protobuf-compiler libncurses5-dev
```

Comment:
The protobuf dependencies are for sawtooth, right? Does the client need these?

Reply:
We need them for the client if the client is for a sawtooth ledger. So until we drop sawtooth, they are necessary.